doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -495,4 +500,42 @@ YAML::
       community-id-seed: 0
 
 
+Stream data
+~~~~~~~~~~~
+
+It is possible to include stream data in application layer events
+
+YAML::
+
+  
+  - eve-log:
+      # ....
+      stream-data: yes # log data as base64
+      stream-data-printable: yes # log printable data
+
 .. _deprecation policy: https://suricata-ids.org/about/deprecation-policy/
+
+
+Conditional verbosity
+~~~~~~~~~~~~~~~~~~~~~
+
+Flows with alerts contains often interesting data and information. It is possible to use the ``full-log-alerted``
+option to increase the verbosity of log on flows with alerts. If activated, protocols like HTTP and TLS will have
+verbose logging if ever the flow has alerts.
+
+For TLS, extended logging is activated for flows with alerts. For HTTP, extended
+logging is activated, full headers are dumped and HTTP body is added.
+
+
+YAML::
+
+  - eve-log:
+      # ....
+      full-log-alerted: printable # log printable data
```

Comment:
not very fond of this option name, as its not self explaining.

Should these things be under the alert type too?

Reply:
In fact, this is triggering extending logging for alerted flows so IMO the place is correct.

Maybe we name it 'extended-logging-alerted-flows' ?